packages/ndk-multilib/build.sh

```diff
@@ -37,11 +37,51 @@ prepare_libs() {
 	fi
 
 	cp $LIBATOMIC/libatomic.a $TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/$SUFFIX/lib/
+
+	cp $LIBATOMIC/libunwind.a $TERMUX_PKG_MASSAGEDIR/$TERMUX_PREFIX/$SUFFIX/lib/
+}
+
+add_cross_compiler_rt() {
+	RT_PREFIX=$NDK/toolchains/llvm/prebuilt/linux-x86_64/lib64/clang/*/lib/linux
+	RT_OPT_DIR=$TERMUX_PREFIX/opt/ndk-multilib/cross-compiler-rt
+	mkdir -p $TERMUX_PKG_MASSAGEDIR/$RT_OPT_DIR
+	cp $RT_PREFIX/* $TERMUX_PKG_MASSAGEDIR/$RT_OPT_DIR || true
 }
 
 termux_step_extract_into_massagedir() {
 	prepare_libs "arm" "arm-linux-androideabi"
 	prepare_libs "arm64" "aarch64-linux-android"
 	prepare_libs "x86" "i686-linux-android"
 	prepare_libs "x86_64" "x86_64-linux-android"
+	add_cross_compiler_rt
+}
+
+termux_step_create_debscripts() {
+	LIBLLVM_VERSION=$(. $TERMUX_SCRIPTDIR/packages/libllvm/build.sh; echo $TERMUX_PKG_VERSION)
+	RT_PATH=$TERMUX_PREFIX/lib/clang/$LIBLLVM_VERSION/lib/android
+	# Install the symlinks.
+	cat <<- POSTINST_EOF > ./postinst
+	#!$TERMUX_PREFIX/bin/sh
+	echo "Installing symlinks to $RT_PATH..."
+	mkdir -p $RT_PATH
+	find $RT_OPT_DIR -type f ! -name "libclang_rt*\$(dpkg --print-architecture)*" -exec ln -sfr "{}" $RT_PATH \;
```

Comment:
Is the `-r` option for `ln` command necessary here? This is a GNU extension and IMO should be avoided if possible.

Reply:
`-r` is not necessary. We can remove it.